### PR TITLE
Fixing many typos in docs

### PR DESCRIPTION
**Pull request by tuanvc**
This PR fixes many typos in docs

### CODE REVIEW

**Comment by mattfarina** [comment]
@tuanvcw Can you run `make docs` on this change. That is why CI is failing. Some docs are autogenerated. The root needs to be fixed and then the other docs generated.

**Comment by tuanvcw** [comment]
Hi @mattfarina 
Thank you very much! I've just run make docs as your suggestion.
Wish you a great day ahead :)

File: docs/chart_template_guide/functions_and_pipelines.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Template functions and pipelines are a powerful way to transform information and
 
 ## Operators are functions
 
-Operators are implemented as functions that return a boolean value. To use `eq`, `ne`, `lt`, `gt`, `and`, `or`, `not` etcetera place the operator at the front of the statement followed by its parameters just as you would a function. To chain multiple operations together, separate individual functions by surrounding them with paranthesis.
+Operators are implemented as functions that return a boolean value. To use `eq`, `ne`, `lt`, `gt`, `and`, `or`, `not` etcetera place the operator at the front of the statement followed by its parameters just as you would a function. To chain multiple operations together, separate individual functions by surrounding them with parenthesis.
```
Comment (hickeyma) [inline docs/chart_template_guide/functions_and_pipelines.md]: parenthesis --> parentheses

**Review by hickeyma — CHANGES_REQUESTED** [review]
Small nit change

File: docs/chart_template_guide/functions_and_pipelines.md (hunk 1 above)
Comment (tuanvcw) [reply docs/chart_template_guide/functions_and_pipelines.md]: Thank you very much, this has been updated as your recommendation.

Comment (hickeyma) [reply docs/chart_template_guide/functions_and_pipelines.md]: @tuanvcw I do not see the changes

Comment (tuanvcw) [reply docs/chart_template_guide/functions_and_pipelines.md]: @hickeyma, sorry I forgot to push the new change.
The patch has just been updated, thanks a lot :smile:

**Review by hickeyma — APPROVED** [review]
LGTM, thanks

File: docs/chart_template_guide/functions_and_pipelines.md (hunk 1 above)
Comment (hickeyma) [reply docs/chart_template_guide/functions_and_pipelines.md]: Not a bother

**Comment by tuanvcw** [comment]
Hi @mattfarina and @hickeyma,
Thanks for your kind reviews, if there's anything else I could do to move this PR forward, please feel free to let me know.
Have a great day :smile:

**Review by mattfarina — APPROVED** [review]

**Pull request merged**